test/e2e/tests/multi_zone_e2e_test.go

```diff
@@ -157,7 +157,7 @@ func testAttachWriteReadDetach(volID string, volName string, instance *remote.In
 			testFile := filepath.Join(a.publishDir, "testfile")
 			err := testutils.WriteFile(instance, testFile, testFileContents)
 			if err != nil {
-				return fmt.Errorf("Failed to write file: %v", err)
+				return fmt.Errorf("Failed to write file: %v", err.Error())
```

Comment:
1. what is the reason we need `err.Error()` here?
2. `.Error()` returns a string so perhaps `%s` or `%q` instead of `%v`?

Reply:
Keep consistency in logging. Right now we use either %w with err or %v with err.Error() when the error is not nil.

Reply:
Sorry, I still don't quite understand what's the context behind the need to add ".Error()" for all err messaging, is there a custom `Error()` function we're preparing?

Reply:
.Error() unwrap and return the error message only, err obj itself printing in %v may contain some other information depending on the error, like error code string erc.